charts/speakeasy-k8s/values.yaml

```
@@ -55,6 +55,9 @@ registry:
   # used by managed speakeasy to send invites to the platform. No need to set this value for selfhosted speakeasy
   # sendGridKey: "**********"
 
+  # An optional field that specifies the number of desired Pods for the registry-web-deployment. It defaults to 1.
+  #  replicas: 1
```

```suggestion
  #  replicas: 2
```

I would say default to 2 as its best practice to have redundancy